play.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: default
+---
+
+<div class="jumbotron jumbotron-fluid play" style="background: url('{{ site.url }}/assets/images/bg.jpg') #2c2f33 no-repeat center center fixed; background-size: cover;">
+    <div class="container">
+        <h1 class="heading">Play</h1>
+        <p class="lead">Play anytime, anywhere</p>
+    </div>
+</div>
+<div class="container">
+    <div class="row">
+        <div class="col-sm-3">
+            <div class="sidebar">
+                <div class="card">
+                    <div class="card-body">
+                        <h5 class="card-title heading">Connect to</h5>
+                        <hr />
+                        <input class="input" onclick="this.select();" readonly="" type="text" value="{{ site.mc_url }}">
```

Comment:
`onclick` should also copy the address to the clipboard. Maybe a clipboard icon should be there too.

Reply:
I think copying to clipboard required js which is why I avoided it originally. Clicking to select was how the OCN website did it.